hack/generate/dockerfile.sh

```diff
@@ -16,6 +16,7 @@ values[VERSION]="$(metadata.get project.version)"
 values[SERVING_VERSION]="$(metadata.get dependencies.serving)"
 values[EVENTING_VERSION]="$(metadata.get dependencies.eventing)"
 values[EVENTING_KAFKA_VERSION]="$(metadata.get dependencies.eventing_kafka)"
+values[KNATIVE_EVENTING_KAFKA_BROKER_VERSION]="$(metadata.get dependencies.eventing_kafka_broker)"
```

Comment:
```suggestion
values[EVENTING_KAFKA_BROKER_VERSION]="$(metadata.get dependencies.eventing_kafka_broker)"
```

Reply:
This is just to unify the approach. The other variables do not have the "KNATIVE" prefix either.